src/site/content/en/blog/how-mercadolibre-optimized-web-vitals/index.md

```diff
@@ -0,0 +1,212 @@
+
+<figure class="w-figure">
+  <img src="lighthouse-unoptimized.png" 
+       alt="Lighthouse metrics in a PSI report for Mercado Libre's product detail pages.">
+</figure>
+
+Based on this, Mercado Libre set a goal to improve their Max Potential FID score in a laboratory tool like Lighthouse and WebPageTest, under the assumption that these improvements would affect their real users, and therefore, show up in real user monitoring tools like the Chrome User Experience Report.
+
+## Optimize long tasks
+
+### First iteration
+
+Based on the main thread trace, Mercado Libre set the goal of optimizing the two modules that were running expensive code.
+
+They started optimizing the performance of the internal tracking module. This module contained a CPU-heavy task that wasn't critical for the module to work, and therefore could be safely removed. This led to a 2% reduction in JavaScript for the whole site.
+
+After that they started to work on **improving the general bundle size**:
+
+Mercado Libre used [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer) to detect opportunities for optimization:
+
+- Initially they were requiring the full [Lodash module](https://lodash.com/). This was replaced with a [per-method require](https://lodash.com/per-method-packages) to load only a subset of Lodash instead of the whole library, and used in conjunction with [lodash-webpack-plugin](https://github.com/lodash/lodash-webpack-plugin) to shrink Lodash even further.
+
+They also applied the following [Babel](https://babeljs.io/) optimizations:
+
+- Using [@babel/plugin-transform-runtime](https://babeljs.io/docs/en/babel-plugin-transform-runtime) to reuse Babel's helpers throughout the code, and reduce the size of the bundle considerably.
+- Using [babel-plugin-search-and-replace](https://github.com/jean-smaug/babel-plugin-search-and-replace#readme) to replace tokens at build time, in order to remove a large configuration file inside the main bundle.
+- Adding [babel-plugin-transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types#readme) to save some extra bytes by removing the prop types.
+
+As a result of these optimizations, the bundle size was reduced **by approximately 16%**.
+
+## Measure impact
+
+The changes lowered Mercado Libre's consecutive long tasks **from two seconds to one second**:
+
+<figure class="w-figure">
+  <img src="main-thread-iteration-1.png" 
+       alt="Main thread view of Mercado Libre's product detail pages after first round of optimizations.">
+   <figcaption class="w-figcaption">
+      In the top waterfall of WPT there’s a long red bar (in the Page is Interactive row) between seconds 2 and 5. In the bottom waterfall, the bar has been broken into smaller pieces, occupying the main thread for shorter periods of time.
+  </figcaption>     
+</figure>
+
+Lighthouse showed a **50% reduction** in Max Potential First Input Delay:
```

Comment:
```suggestion
Lighthouse showed a **57% reduction** in Max Potential First Input Delay:
```
We should use the precise number. 1710ms to 750ms. I think that's 57% but you should double-check my math.